pkg/controllers/rollout/controller_test.go

```diff
@@ -888,8 +782,13 @@ func TestPickBindingsToRoll(t *testing.T) {
 				generateClusterResourceBinding(fleetv1beta1.BindingStateScheduled, "snapshot-1", cluster1),
 			},
 			latestResourceSnapshotName: "snapshot-2",
-			crp:                        crpWithApplyStrategy,
-			wantTobeUpdatedBindings:    []int{0},
```

Comment:
this one is 
"test pick scheduled outdated bindings to latest crp apply strategy"

Reply:
btw, since the name says "bindings", why not create more than one?